animation.js

```diff
@@ -160,18 +160,22 @@ function runAnimationStep (time, start, callback, context)
     }
 }
 
-
+/**
+ * Stops any animation on the given element
+ *
+ * @param {HTMLElement} element
+ */
 export function stopAnimation (element)
 {
-    if (typeof element._currentAnimation !== "undefined" && element._currentAnimation !== null)
+    // check for undefined + null
+    if (element._currentAnimation != null)
```

Comment:
Isn't ESLint complaining here?

Reply:
No, we allow lenient checks for `undefined` + `null`